### ISSUE TITLE
logging: API to access the current rpmlogCallbackData

### ISSUE THREAD

**Issue by lukash** [rpm-software-management/rpm#1988]
The `rpmlogSetCallback()` function returns the current `rpmlogCallback`, but there is no way to retrieve the current `rpmlogCallbackData`. The returned callback is useless without its corresponding data, as those need to go together.

There should be a way on the API to retrieve the current data.

We'll quite likely depend on this for dnf 5.

**Comment by pmatilai** [comment]
Yup, this is a common flaw in probably all rpm callback APIs. Should be easy fix to add a proper getter API for them all, just hasn't been demand before now.
